policyengine_us/variables/gov/states/la/tax/income/credits/cdcc/la_cdcc_non_refundable.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+import numpy as np
+
+
+class la_cdcc_non_refundable(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana non refundable cdcc"
+    unit = USD
+    definition_period = YEAR
+    reference = "http://legis.la.gov/Legis/Law.aspx?d=101769"
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.la.tax.income.credits
+        # determine AGI eligibility
+        us_agi = tax_unit("adjusted_gross_income", period)
+        agi_eligible = us_agi > p.cdcc.agi_threshold
+        # determine LA nonrefundable cdcc amount
+        us_cdcc = tax_unit("cdcc", period)
+        la_non_refundable_cdcc = us_cdcc * p.cdcc.non_refundable.rate.calc(
+            us_agi
+        )
+        la_non_refundable_cdcc = np.where(
+            us_agi > p.cdcc.non_refundable.upper_bracket.income_threshold,
+            np.minimum(
```

Comment:
separate the statement out:
upper_bracket = agi > p.upper_bracket.income_threshold
upper_bracket_amount = min_(p.upper_bracket.max_amount, la_non_refundable_cdcc_pre_cap)
la_non_refundable_cdcc = where(upper_bracket, upper_bracket_amount, la_non_refundable_cdcc_pre_cap)

Reply:
comment the statement out